pxd.c

```diff
@@ -132,8 +143,11 @@ static void pxd_release(struct gendisk *disk, fmode_t mode)
 	BUG_ON(pxd_dev->magic != PXD_DEV_MAGIC);
 	pxd_dev->open_count--;
 	spin_unlock(&pxd_dev->lock);
-
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6,5,0)
+	trace_pxd_release(pxd_dev->dev_id, pxd_dev->major, pxd_dev->minor);
```

Comment:
Or just get rid of it. I doubt this is ever going to be used.

Reply:
True,  not clear when tracing is enabled or how we enable it so worked around it...  thx